pkg/asset/installconfig/alibabacloud/validation.go

```diff
@@ -100,6 +119,56 @@ func validateResourceGroup(client *Client, ic *types.InstallConfig, path *field.
 	return append(allErrs, field.NotFound(path.Child("resourceGroupID"), ic.AlibabaCloud.ResourceGroupID))
 }
 
+func validateVpc(client *Client, ic *types.InstallConfig, path *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	vpcs, err := client.ListVpcs(ic.AlibabaCloud.VpcID)
+	if err != nil {
+		return append(allErrs, field.InternalError(path.Child("vpcID"), err))
+	}
+	if vpcs.TotalCount == 1 {
+		return allErrs
+	}
+	return append(allErrs, field.NotFound(path.Child("vpcID"), ic.AlibabaCloud.VpcID))
+}
+
+func validateVSwitchs(client *Client, ic *types.InstallConfig, path *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if ic.AlibabaCloud.VpcID == "" {
```

Comment:
This should be checked via static validation in pkg/types/alibabacloud/validation/platform.go.